content/general/concepts/semantic-technologies/terms/rdf.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: 'rdf'
+Description: 'A language for representing semantic data.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Data'
+  - 'Semantic
+  - 'Metadata'
+  - 'Tuples'
+CatalogContent:
+  - 'paths/data-science-foundations'
+  - 'paths/data-science-nlp'
+---
+
+**Resource Description Framework (RDF)** is a language for representing information. All data within the suite of semantic technologies is represented in RDF.
+
+## RDF Triples
+
+RDF represents data in the format of **triples** -- ordered sets of three elements. The elements in an RDF triple stand for a subject, a predicate, and an object. An RDF triple can therefore be represented as follows:
+
+```pseudo
+subject, predicate, object
+```
+
+- *`subject`*: The element which the triple says something about.
+- *`predicate`*: A feature or property that applies to `subject`.
+- *`object`*: The value of the feature or property represented by `predicate`.
+
+To illustrate, take the sentence 'The novel *Pachinko* was written by Min Jin Lee'. The sentence conveys something about *Pachinko*: that it has a feature of having an author, and that this author is Min Jin Lee. This information can be represented by the following triple:
```

Comment:
```suggestion
As an illustration, consider the sentence 'The novel Pachinko was written by Min Jin Lee.' This sentence communicates a detail about Pachinko: specifically, that it possesses a characteristic of having an author, and that this author is Min Jin Lee. This information can be depicted using the following triple:
```

Reply:
I used italics for 'Pachinko' as it is a convention to put book titles in italics.